README.md

```diff
@@ -0,0 +1,2124 @@
+
+## 7.4. Using Distributed Data Structures
+
+Most of the distributed data structures are supported by the Python client. In this chapter, you will learn how to use these distributed data structures.
+
+### 7.4.1. Using Map
+
+Hazelcast Map is a distributed dictionary. Through the Python client, you can perform operations like reading and writing from/to a Hazelcast Map with the well known get and put methods. For details, see the [Map section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#map) in the Hazelcast IMDG Reference Manual.
+
+A Map usage example is shown below.
+
+```python
+my_map = client.get_map("my-distributed-map").blocking()
+
+my_map.put("key", "value") # Standard put and get
+my_map.get("key")
+
+my_map.put_if_absent("somekey", "somevalue") # Concurrent map methods, optimistic updating
+my_map.replace_if_same("key", "value", "newvalue")
+```
+
+### 7.4.2. Using MultiMap
+
+Hazelcast MultiMap is a distributed and specialized map where you can store multiple values under a single key. For details, see the [MultiMap section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#multimap) in the Hazelcast IMDG Reference Manual.
+
+A MultiMap usage example is shown below.
+
+```python
+multi_map = client.get_multi_map("my-distributed-multimap").blocking()
+
+multi_map.put("my-key", "value1") # Put values in the map against the same key
+multi_map.put("my-key", "value2")
+multi_map.put("my-key", "value3")
+
+values = multi_map.get("my-key") # Print out all the values for associated with key called "my-key"
+
+print(values) # Outputs '['value2', 'value1', 'value3']'
+
+multi_map.remove("my-key", "value2") # Remove specific key/value pair
+```
+
+### 7.4.3. Using Replicated Map
+
+Hazelcast ReplicatedMap is a distributed key-value data structure where the data is replicated to all members in the cluster. It provides full replication of entries to all members for high speed access. For details, see the [Replicated Map section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#replicated-map) in the Hazelcast IMDG Reference Manual.
```

Comment:
Hazelcast Replicated Map ...

NOTE: I will open a PR requesting the same change for all the other client's ReadMe documents.